DigitalLearningSolutions.Web/ViewModels/MyAccount/MyAccountViewModel.cs

```diff
@@ -23,12 +24,22 @@ public class MyAccountViewModel
             DelegateNumber = delegateUser?.CandidateNumber;
             AliasId = delegateUser?.AliasId;
             JobGroup = delegateUser?.JobGroupName;
+            ProfessionalRegistrationNumber = delegateUser?.HasBeenPromptedForPrn == true

```

Comment:
Can you get rid of the `== true`? You might need a `.Value`. That's the more conventional way of doing it when you have a bool variable.

Reply:
its telling me .Value doesn't exist on HasBeenPrompted and not letting me do .Equals because it could be null as delegateUser is potentially null

Reply:
Oh, I see, HasBeenPromptedForPrn isn't nullable, it's because delegateUser is nullable. Okay, that's fine.